homeassistant/components/knx/websocket.py

```diff
@@ -72,14 +74,39 @@ def ws_info(
     connection.send_result(
         msg["id"],
         {
-            "version": knx.xknx.version,
+            "xknx_version": knx.xknx.version,
+            "xknxproject_version": xknxproject_version,
```

Comment:
I guess we don't need to know the current installed xknxproject version, but the one used to parse the project. This is stored in the resulting json in `info/xknxproject_version`.

Reply:
ACK - will revert the changes